Bug: Angular app, Cannot find modules ObservablePoint and PointData in node_modules/pixi.js/lib

### Current Behavior

Hi guys. 
I have a problem with pixi.js v8.9.0

I'm use Ionic + Angular + Pixi.js and packages installed from npm. Simple blank Ionic angular app only with three lines of code:
```
    const app = new Application();
    await app.init({width: 640, height: 360});

    this.renderer.appendChild(this.gameContainer?.nativeElement, app.canvas);
```

When I tried to run **ng serve** I got this errors:
```
X [ERROR] TS2307: Cannot find module '~/maths/point/PointData' or its corresponding type declarations. [plugin angular-compiler]

    node_modules/pixi.js/lib/scene/sprite-nine-slice/NineSliceGeometry.d.ts:2:31:
      2 │ import { type PointData } from '~/maths/point/PointData';
        ╵                                ~~~~~~~~~~~~~~~~~~~~~~~~~


X [ERROR] TS2307: Cannot find module '~/maths/point/ObservablePoint' or its corresponding type declarations. [plugin angular-compiler]

    node_modules/pixi.js/lib/scene/sprite-nine-slice/NineSliceSprite.d.ts:3:32:
      3 │ import { ObservablePoint } from '~/maths/point/ObservablePoint';
        ╵                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


X [ERROR] TS2307: Cannot find module '~/maths/point/PointData' or its corresponding type declarations. [plugin angular-compiler]

    node_modules/pixi.js/lib/scene/sprite-nine-slice/NineSliceSprite.d.ts:4:31:
      4 │ import { type PointData } from '~/maths/point/PointData';
        ╵                                ~~~~~~~~~~~~~~~~~~~~~~~~~

```

Errors exists at v8.9.0 and v8.8.0. Version 8.7.3 works fine.

For quick fix, I'm directly edited .d.ts files import paths in the lib folder:
```
import { ObservablePoint } from '../../maths/point/ObservablePoint';
import { type PointData } from '../../maths/point/PointData';
```
And app runs fine for now, but this is not proper fix unfortunately. What I should to do? Rollback to 8.7.3 version or work with quick-fixed 8.9.0 or need to install pixi in some another way?

### Expected Behavior

**ng serve** command run app without errors 

### Steps to Reproduce

install ionic cli v7.0+
run **ionic new** and make blank Angular app. 
npm i pixi.js

replace code in the main.page.html to:
`<div style="flex: 1" #gameContainer></div>`

replace code in the main.page.ts to:
```
export class HomePage implements AfterViewInit {

  renderer = inject(Renderer2);

  @ViewChild('gameContainer', {
    read: ElementRef,
    static: true,
  }) gameContainer: ElementRef | undefined;

  async ngAfterViewInit() {
    const app = new Application();
    await app.init({width: 640, height: 360});

    this.renderer.appendChild(this.gameContainer?.nativeElement, app.canvas);
  }
}
```

then try to run **ng serve** or **ionic serve** 



### Environment

pixi.js: 8.9.0
Ionic cli: 7.2.1 
ionic/angular: 8.0.0
Angular: 19.0  
Standalone angular app 

### Possible Solution

_No response_

### Additional Information

_No response_

--- a/src/rendering/renderers/shared/shader/Shader.ts
+++ b/src/rendering/renderers/shared/shader/Shader.ts
@@ -1,10 +1,10 @@
 import EventEmitter from 'eventemitter3';
+import { uid } from '../../../../utils/data/uid';
 import { GlProgram } from '../../gl/shader/GlProgram';
 import { BindGroup } from '../../gpu/shader/BindGroup';
 import { GpuProgram } from '../../gpu/shader/GpuProgram';
 import { RendererType } from '../../types';
 import { UniformGroup } from './UniformGroup';
-import { uid } from '~/utils/data/uid';
 
 import type { GlProgramOptions } from '../../gl/shader/GlProgram';
 import type { BindResource } from '../../gpu/shader/BindResource';

--- a/src/scene/sprite-nine-slice/NineSliceGeometry.ts
+++ b/src/scene/sprite-nine-slice/NineSliceGeometry.ts
@@ -1,5 +1,5 @@
+import { type PointData } from '../../maths/point/PointData';
 import { PlaneGeometry } from '../mesh-plane/PlaneGeometry';
-import { type PointData } from '~/maths/point/PointData';
 
 /**
  * Options for the NineSliceGeometry.

--- a/src/scene/sprite-nine-slice/NineSliceSprite.ts
+++ b/src/scene/sprite-nine-slice/NineSliceSprite.ts
@@ -1,9 +1,9 @@
+import { ObservablePoint } from '../../maths/point/ObservablePoint';
+import { type PointData } from '../../maths/point/PointData';
 import { Texture } from '../../rendering/renderers/shared/texture/Texture';
 import { deprecation, v8_0_0 } from '../../utils/logging/deprecation';
 import { ViewContainer, type ViewContainerOptions } from '../view/ViewContainer';
 import { NineSliceGeometry } from './NineSliceGeometry';
-import { ObservablePoint } from '~/maths/point/ObservablePoint';
-import { type PointData } from '~/maths/point/PointData';
 
 import type { Size } from '../../maths/misc/Size';
 import type { View } from '../../rendering/renderers/shared/view/View';

